src/structs.md

```diff
@@ -17,3 +17,17 @@ fn main() {
     println!("{} is {} years old", peter.name, peter.age);
 }
 ```
+
+<details>
+
+* The top of the program is how we define a struct, the main function creates a struct, and then at the bottom we access the fields.
```

Comment:
I think we can state this in a shorter way: the notes should not be a "script" for the instructor. The instructor is expected to already know Rust well, so we should instead tell the instructor what to show on each page —we should not teach them Rust.